### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Walkman100**
Updated Debian Linux commands with Syntax Highlighting

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,19 +82,27 @@ It downloads the latest version of ZeroNet then starts it automatically.
...
-* Open http://127.0.0.1:43110/ in your browser and enjoy! :)
+```sh
+sudo apt-get update
+sudo apt-get install msgpack-python python-gevent
+wget https://github.com/HelloZeroNet/ZeroNet/archive/master.tar.gz
+tar xvpfz master.tar.gz
+cd ZeroNet-master
+# to start:
+python zeronet.py
+xdg-open http://127.0.0.1:43110/
```
Comment (Walkman100) [inline README.md]: I've just ran this, and realised that it stays running - i.e., you can't execute `xdg-open` because `python zeronet.py` stays running.

**Comment by Walkman100** [comment]
bump

**Comment by HelloZeroNet** [comment]
Thanks, but I think it's not improves the readability (syntax highlighting only works on cd command)

**Comment by Walkman100** [comment]
I thought it was easier to read as a code block without the white in-between

**Comment by TheNain38** [comment]
Yeah, me too ^^

**Pull request closed without merging**